flake.nix

```diff
@@ -54,101 +56,51 @@
     treefmt-nix.url = "github:numtide/treefmt-nix";
   };
 
-  outputs =
-    inputs@{ flake-parts, self, ... }:
-    flake-parts.lib.mkFlake { inherit inputs; } {
-      systems = import inputs.systems;
-
-      imports = [
-        ./modules
-        inputs.lite-config.flakeModule
-        inputs.treefmt-nix.flakeModule
-      ];
-
-      lite-config =
-        { lib, ... }:
-        {
-          nixpkgs = {
-            config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [ "terraform" ];
-            overlays = [
-              (final: prev: {
-                hydra = (prev.hydra.override { nix = final.nixVersions.nix_2_24; }).overrideAttrs (o: {
-                  version = inputs.hydra.shortRev;
-                  src = inputs.hydra;
-                  buildInputs = o.buildInputs ++ [ final.perlPackages.DBIxClassHelpers ];
-                });
-              })
-            ];
-          };
+  # overlays = [
+  #   (final: prev: {
+  #     hydra = (prev.hydra.override { nix = final.nixVersions.nix_2_24; }).overrideAttrs (o: {
+  #       version = inputs.hydra.shortRev;
+  #       src = inputs.hydra;
+  #       buildInputs = o.buildInputs ++ [ final.perlPackages.DBIxClassHelpers ];
+  #     });
+  #   })
+  # ];
 
-          hostModuleDir = ./hosts;
+  # treefmt = {
+  #   flakeCheck = system == "x86_64-linux";
+  #   imports = [ ./dev/treefmt.nix ];
+  # };
```

Comment:
You can create a `package/formatter/default.nix` for this.

See https://github.com/zimbatm/home/blob/main/packages/formatter/default.nix